### ISSUE TITLE
Couldn't connect to Docker daemon

### ISSUE THREAD

**Issue by k2xl** [jpetazzo/dind#56]
I have a vagrant machine. Trying to run:
`docker run -privileged -d -p 1234 -e PORT=1234 jpetazzo/dind`

I then try docker version and I see
```
Couldn't connect to Docker daemon at http+unix://var/run/docker.sock - is it running?               │        ../docker-heka/
                                                                                                    │        ../mariadb/
If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.    ```

**Comment by vdemeester** [comment]
If you want to connect from your host in the engine in the docker run something like this:

```bash
$ docker run --privileged -d -p 1234:1234 -e PORT=1234 jpetazzo/dind
# To connect to it
$ DOCKER_HOST=127.0.0.1:1234 docker ps
```

It should work fine. Note that the actual images published in the hub is based on docker 1.4.1, so if you are using docker 1.5 on your host, it won't work.. :-/

**Comment by jpetazzo** [comment]
I will close this issue. If the OP or someone else still has the same problem, feel free to comment and I will reopen.

Also — I have updated the `jpetazzo/dind` image to the latest Docker release, and I also have updated the repo links so that it would track it more closely! :-)
